app/tetwild/Parameters.h

```diff
@@ -6,22 +6,32 @@
 #include "common.h"
 
 namespace tetwild {
-struct Parameters
-{
-    double epsr;
+struct Parameters {
+    double epsr = 1 / 1000.;
     double eps;
-    double lr;
-    double l;
+    double lr = 1 / 20.;
+    double l = -1;
     double diag_l;
     Vector3f min, max;
 
     double splitting_l2;
     double collapsing_l2;
 
-    void init()
-    {
+    void init(const Vector3f &min_, const Vector3f &max_) {
```

Comment:
The original `init()` is missing, and some tests are failing because of it.

Reply:
Also, I am still quite confused with the `Vector3f` here, why is it typedef as `Eigen::Vector3d`, is it a typo?